model/src/main/java/org/mskcc/cmo/metadb/model/SampleMetadata.java

```diff
@@ -101,7 +102,7 @@ public SampleMetadata(String igoId, String cmoInfoIgoId, String cmoSampleName, S
             String tumorOrNormal, String tissueLocation, String specimenType, String sampleOrigin,
             String preservation, String collectionYear, String sex, String species, String tubeId,
             String cfDNA2dBarcode, List<QcReport> qcReports, List<Library> libraries,
-            String mrn, String cmoSampleId, String sampleType, String tumorType, String parentTumorType,
```

Comment:
`cmoSampleId` wasn't being set or used anywhere so I removed it. I think this might have been added by accident to be used as `cmoSampleName` (the generated CMO sample label)